src/js-setup.js

```
@@ -94,7 +94,17 @@ JsSetup.prototype.init = function (opts) {
 			var myFtOpts = opts.myFtOpts || {};
 			myFtOpts.follow = flags.get('follow');
 			myFtOpts.saveForLater = flags.get('saveForLater');
-			myFtClient.init(myFtOpts);
+
+			myFtClient.init(myFtOpts)
+				.catch(function (err) {
+
+					if(err.message === 'No session cookie found') {
+						//if user is anonymous, swallow this error to avoid annoying console logs
+						return;
+					}
+
+					throw err;
```

My understanding is that if it's the session cookie error, it'll get swallowed, but if it's any other error it'll get thrown within the promise like before, as if the catch block was never there (a bit of fiddling: http://www.es6fiddle.net/ielbjhq1/)